cypress/e2e/GoogleSearchResult/documentation.cy.js

```diff
@@ -0,0 +1,96 @@
+import { GOOGLE_SEARCH_RESULT_BASE_URL, APILAYER_BASE_URL } from '../../resources/data';
+
+describe('Google search result API documentation page', () => {
+    before(() => {
+        cy.visit(GOOGLE_SEARCH_RESULT_BASE_URL);
+    });
+    
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+  
+    it('2. Test to check "Documentation" tab visibility', () => {
+        cy.componentVisiblityCheck('#documentation-tab');
+    });
+  
+    it('3. Test to click "Documentation" tab', () => {
+        cy.get('#documentation-tab').click();
```

Comment:
add apirequest api interupt here too

Reply:
Add API interupt wait here.
Also the tab has an id use that to get the element